.example.env

```diff
@@ -0,0 +1,3 @@
+# create an .env.local file with proper values:
+
+GOOGLE_MAP_API_KEY=""
```

Comment:
@langleynick i added an example env file, don't put the files here, but this will mirror the fields. 

it will be a source of truth for actual `.env.local` files.